contracts/Parameters/ParameterManagerStorage.sol

```diff
@@ -48,6 +48,7 @@ abstract contract ParameterManagerStorageV1 is IParameterManager {
 
 abstract contract ParameterManagerStorageV2 is ParameterManagerStorageV1 {
     IERC20Upgradeable public nativeWrappedToken;
+    uint256 public freeReactionLimit;
```

Comment:
We should probably put ///@dev comments here explaining what the var is... I forgot to do it on `nativeWrappedToken` right above :(

A new dev might not understand what the vars are meant to hold.

Reply:
done